packages/dom-expressions/src/constants.js

```diff
@@ -1,16 +1,18 @@
+// list of lowercase booleans
 const booleans = [
   "allowfullscreen",
   "async",
-  "autofocus",
+  "alpha", // HTMLInputElement
+  "autofocus", // HTMLElement prop
   "autoplay",
   "checked",
   "controls",
   "default",
   "disabled",
   "formnovalidate",
-  "hidden",
+  // "hidden", // HTMLElement prop - not a boolean
```

Comment:
It's not?
Looking at the attribute it looks like it works like a boolean attribute: https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Global_attributes/hidden. In that it's presence sets it to true.

Looking at the property: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/hidden
It looks like it takes booleans and or a string but that should still work on property set.

Does empty string mess with it in a weird way? It looks like it is operating fine as is: https://playground.solidjs.com/anonymous/7c281ee5-b35b-45c3-90a0-32c6da8fd7f8. Atleast consistent with how the others work like disabled.